packages/features/ee/organizations/components/OrganizationAvatar.tsx

```diff
@@ -15,7 +15,7 @@ const OrganizationAvatar = ({ size, imageSrc, alt, organizationSlug, ...rest }:
       indicator={
         organizationSlug ? (
           <div
-            className={classNames("absolute bottom-0 right-0 z-10", size === "lg" ? "h-3 w-3" : "h-10 w-10")}>
+            className={classNames("absolute bottom-0 right-0 z-10", size === "lg" ? "h-6 w-6" : "h-10 w-10")}>
```

Comment:
Found the sub-avatar for large regular avatar excessively small, increasing its size